.github/workflows/release.yaml

```
@@ -0,0 +1,29 @@
+name: "Release"
+
+on: workflow_dispatch
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: create branch and tag
+        run: |
+          RELEASE_BRANCH=$(./.github/scripts/release-branch-name.sh)
+          OLD_VERSION=$(git describe --abbrev=0 --tags)
+          NEW_VERSION=$(./.github/scripts/new-tag.sh)
```

```suggestion
          OLD_VERSION=$(git describe --abbrev=0 --tags)
          IFS=. read -r major minor patch <<< "$OLD_VERSION"
          RELEASE_BRANCH="release-$major.$((++minor))" 
          NEW_VERSION="$major.$((++minor)).0-rc1"
```

and no need for the scripts?